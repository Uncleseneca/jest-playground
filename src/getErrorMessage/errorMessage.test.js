import getErrorMessage from './errorMessage'

describe('errorMessage', () => {
  it('returns an error message for valid code', () => {
    expect(getErrorMessage(1)).toMatchSnapshot()
    expect(getErrorMessage(2)).toMatchSnapshot()
    expect(getErrorMessage(3)).toMatchSnapshot()
  })

  it('throws an error if the code is not valid', () => {
    expect(() => {
      getErrorMessage(4)
    }).toThrowErrorMatchingSnapshot()
  })
})
