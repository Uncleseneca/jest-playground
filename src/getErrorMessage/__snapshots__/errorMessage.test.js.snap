// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`errorMessage returns an error message for valid code 1`] = `"The camel walks on a leg"`;

exports[`errorMessage returns an error message for valid code 2`] = `"Rabbits don't eat carrots"`;

exports[`errorMessage returns an error message for valid code 3`] = `"Cats don't eat mouses"`;

exports[`errorMessage throws an error if the code is not valid 1`] = `"No error messages for that code"`;
